docker/quarto/Dockerfile

```diff
@@ -0,0 +1,36 @@
+ARG VAST_VERSION
+ARG QUARTO_VERSION
+
+FROM tenzir/vast:$VAST_VERSION
+ARG QUARTO_VERSION
+ARG TARGETPLATFORM
+ARG TARGETARCH
+
+USER root
+
+RUN apt-get update && \
+    apt-get install -y \
+        r-base \
+        python3-pip \
+        git \
+        curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update && \
+    apt-get install -y python3-pip curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN git clone -b v${QUARTO_VERSION} https://github.com/quarto-dev/quarto-cli.git /opt/quarto && \
+    cd /opt/quarto && \
+    ./configure.sh
+
+USER vast:vast
+
+# We install Poetry alongside VAST, they shouldn't disturb each other :)
```

Comment:
```suggestion
```

I think it's obvious what we're doing here. Smileys always compromise a sense of professionalism, which I would prefer to avoid in comments. This is separate from discussion. 😉

Reply:
I disagree that it's obvious, given that $PREFIX is defined in the base image... duly noted for smileys, but with a pinch in my heart as I really like them 😄